test/release/examples/patterns/metadata-20lines.csv

```diff
@@ -0,0 +1,21 @@
+cord_uid,sha,source_x,title,doi,pmcid,pubmed_id,license,abstract,publish_time,authors,journal,mag_id,who_covidence_id,arxiv_id,pdf_json_files,pmc_json_files,url,s2_id
```

Comment:
Maybe I am coming to this out of context, but are these other 2 files supposed to be in this PR?